### PR TITLE
Map: Implement new design and responsiveness

### PR DESCRIPTION
**Pull request by s0**
## Description

(opening draft now so that we can deploy this branch and people can play around with it while it's still in-development)

Implement the new design and responsive functionality for the map.

## Release Checklist

- [ ] This code has unit tests
- [x] I have a plan to verify that these changes are working as expected after deploy
- [x] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Additional Notes

Closes: #75

### CODE REVIEW

**Comment by puzzledbytheweb** [comment]
Shit. I clicked "Ready for Review" because I read "Ready for Preview" and hoped for some reason that it would direct me to the preview of the map -.- 

Can you undo that? @s0

**Comment by s0** [comment]
> Shit. I clicked "Ready for Review" because I read "Ready for Preview" and hoped for some reason that it would direct me to the preview of the map -.- 
> 
> Can you undo that? @s0  

Haha, no there's no way to re-draft it unfortunately, that's okay we'll just keep it open! :-D

**Comment by s0** [comment]
Okay, i think enough of the design is implemented now that we can get this merged and deployed. the experience is already significantly better on mobile, and the only parts of the design that aren't done are the results panel, and the mobile filters.

File: map/src/styling/index.ts [hunk 1]
```diff
@@ -14,8 +14,23 @@ const {
 export { css, createGlobalStyle, keyframes, ThemeProvider, THEME };
 export default styled;
 
+const LARGE_BREAKPOINT = 1000;
```
Comment (puzzledbytheweb) [inline map/src/styling/index.ts:17]: We have a file here with all the breakpoints, maybe we can derive a way to use this instead? - https://github.com/reach4help/reach4help/blob/master/web-client/src/constants/mediaQueries.ts

Comment (s0) [reply map/src/styling/index.ts:17]: Ah, sharing between `web-client` and `map` requires introducing a new package with a compile step (or using symlinks). I think it's something we want to do, but probably shoudln't block this going in based on sharing of these constants (in particular given that the breakpoints are different right now).

Comment (puzzledbytheweb) [reply map/src/styling/index.ts:17]: Yeah, I agree. I'll open up a new issue though so we are aware of this later

**Pull request merged**